resources/js/core.js

```diff
@@ -154,6 +155,7 @@ const photoBooth = (function () {
             $('.loading').text(L10N.cheese);
         } else {
             $('.loading').text(L10N.cheeseCollage);
+            $('<p>').text(`${nextCollageNumber + 1} / ${collageLimit}`).appendTo('.loading');
```

Comment:
`${result.limit}` instead ?

Reply:
Nope, that won't work unfortunately as `result` is no global variable and hence not available within that scope. Mid term both this newly introduced `collageLimit` as well as `result.limit` (which is in turn hard-coded `4` in `takePic.php` should be put into `lib/config.php` (I guess).

Reply:
Mh... Ok
Putting it in config.php would mean we need to allow different collage options (e.g. collage out of 2 pictures, out of 3 pictures etc.). This means the whole collage function needs to be reworked to always generate a collage.

Reply:
I probably put it wrong... :) I wasn't about to suggest it should be configurable but it should be available as a global/static/constant variable.